alicloud/alicloud_cs_kubernetes_cluster.go

```diff
@@ -0,0 +1,481 @@
+				Transform:   transform.FromField("vpc_id"),
+			},
+			{
+				Name:        "vswitch_id",
+				Description: "The IDs of VSwitches.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("vswitch_id"),
+			},
+			{
+				Name:        "vswitch_cidr",
+				Description: "The CIDR block of VSwitches.",
+				Type:        proto.ColumnType_CIDR,
+				Transform:   transform.FromField("vswitch_cidr"),
+			},
+			{
+				Name:        "data_disk_category",
+				Description: "The type of data disks.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("data_disk_category"),
+			},
+			{
+				Name:        "data_disk_size",
+				Description: "The size of a data disk.",
+				Type:        proto.ColumnType_INT,
+				Transform:   transform.FromField("data_disk_size"),
+			},
+			{
+				Name:        "zone_id",
+				Description: "The ID of the zone where the cluster is deployed.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("zone_id"),
+			},
+			{
+				Name:        "network_mode",
+				Description: "The network type of the cluster.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("network_mode"),
+			},
+			{
+				Name:        "subnet_cidr",
+				Description: "The CIDR block of pods in the cluster.",
+				Type:        proto.ColumnType_CIDR,
+				Transform:   transform.FromField("subnet_cidr"),
+			},
+			{
+				Name:        "external_loadbalancer_id",
+				Description: "The ID of the Server Load Balancer (SLB) instance deployed in the cluster.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("external_loadbalancer_id"),
+			},
+			{
+				Name:      "port",
+				Type:      proto.ColumnType_STRING,
+				Transform: transform.FromField("port"),
+			},
+			{
+				Name:        "node_status",
+				Description: "The status of cluster nodes.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("node_status"),
+			},
+			{
+				Name:        "cluster_healthy",
+				Description: "The health status of the cluster.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("cluster_healthy"),
+			},
+			{
+				Name:        "docker_version",
+				Description: "The version of Docker.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("docker_version"),
+			},
+			{
+				Name:      "swarm_mode",
+				Type:      proto.ColumnType_STRING,
+				Transform: transform.FromField("swarm_mode"),
+			},
+			{
+				Name:      "gw_bridge",
+				Type:      proto.ColumnType_STRING,
+				Transform: transform.FromField("gw_bridge"),
+			},
+			{
+				Name:      "upgrade_components",
+				Type:      proto.ColumnType_STRING,
+				Transform: transform.FromField("upgrade_components"),
+			},
+			{
+				Name:      "next_version",
+				Type:      proto.ColumnType_STRING,
+				Transform: transform.FromField("next_version"),
+			},
```

Comment:
Can you add a description for these columns?

Reply:
as discussed , proper description is not available for these columns